testutils/metrics/metrics.go

```diff
@@ -35,9 +35,11 @@ type TestStatsReporter struct {
 	GaugesWG   sync.WaitGroup
 	TimersWG   sync.WaitGroup
 
-	Counters map[string]int64
-	Gauges   map[string]float64
-	Timers   map[string]time.Duration
+	Counters           map[string]int64
+	Gauges             map[string]float64
+	Timers             map[string]time.Duration
+	HistogramSamples   map[string]tally.Buckets
```

Comment:
This is needed only to implement interfaces.

The way service is using it is in service_setup.go, where options became a separate struct.